cmd/main.go

```diff
@@ -30,6 +30,7 @@ const (
 	specialRulesFlagName       = "add-special-rule"
 	ignoreOnExitFlagName       = "ignore-on-exit"
 	maxTargetMegabytesFlagName = "max-target-megabytes"
+	excludedFilesFlagName      = "exclude"
```

Comment:
@yrachelevi What do you think about the name of this argument?

To get a context, here is the other "exclude" options we already have:

```
--add-special-rule strings      special (non-default) rules to apply.
                                This list is not affected by the --rule and --ignore-rule flags.
--ignore-result strings         ignore specific result by id
--ignore-rule strings           ignore rules by name or tag
--rule strings                  select rules by name or tag to apply to this scan
```

Also, in [Kics](https://docs.kics.io/latest/commands/#scan_command_options) we are using `--exclude-paths` for this:

```
-e, --exclude-paths strings         exclude paths from scan
                                      supports glob and can be provided multiple times or as a quoted comma separated string
                                      example: './shouldNotScan/*,somefile.txt'
```

So I think taking the argument name from Kics is good.

@Tzurrr Note that in Kics we accept glob pattern, I think the code is here: https://github.com/Checkmarx/kics/blob/83c6fe181ecbe691f175c8e9c07d7a68c2417fc4/pkg/engine/provider/filesystem.go#L57-L93